spec/tests/osm_material_spec.rb

```diff
@@ -36,6 +36,12 @@ def load_file(file_name)
   file = File.join(File.dirname(__FILE__), '../samples/osm_material/', file_name)
   vt = OpenStudio::OSVersion::VersionTranslator.new
   osm = vt.loadModel(file)
+  year = 2020
+  osm.get.getYearDescription.setCalendarYear(year.to_i)
+  weather_file = File.join(File.dirname(__FILE__), '../samples/epw')
+  workflow = OpenStudio::WorkflowJSON.new
+  workflow.setSeedFile(file)
+  workflow.setWeatherFile(File.absolute_path(weather_file))
```

Comment:
@joseph-robertson I am setting the calendar year and also setting the weather file here. I am still seeing the error: 

`[openstudio.model.YearDescription] <1> 'UseWeatherFile' is selected in YearDescription, but there are no 
weather file set for the model.`

From the code link you shared - this should not be the case. Do you have any thoughts on what could be happening?

Reply:
@tanushree04

(1) Where are you setting the calendar year? Here is a snippet for setting it: https://github.com/NREL/resstock/blob/develop/resources/measures/HPXMLtoOpenStudio/resources/location.rb#L65-L68

(2) Above you are setting the weather file on the osw, and not the model. Here is a snippet for setting it: https://github.com/NREL/resstock/blob/develop/resources/measures/HPXMLtoOpenStudio/resources/location.rb#L37-L45

Reply:
@joseph-robertson thanks for sharing this.
After loading my openstudio model 

1) I am setting the calendar year here: https://github.com/ladybug-tools/honeybee-openstudio-gem/pull/251/files#diff-d7d4c83dbf9adb63b5ff4c9e87720d7754ec1dbad33876ad970dc892313455cbR47
Does this look correct? The resultant model does show 2020 as the calendar year

2) I am not setting the weather file to the model : https://github.com/ladybug-tools/honeybee-openstudio-gem/pull/251/files#diff-d7d4c83dbf9adb63b5ff4c9e87720d7754ec1dbad33876ad970dc892313455cbR51

Reply:
@tanushree04 Does your simulation still error out?

Reply:
Yes, it is still printing out the warnings.
The command I use to run it is `bundle exec rake` which is running all the spec files.